python-kubernetes/rookout-service/README.md

```diff
@@ -2,6 +2,10 @@
 
 This is a full example for deploying an application with Rookout using Kubernetes.
 
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
```

Comment:
? I updated it a week ago

Reply:
+ What does it have to do with serverless?

Reply:
It doesn't 
I just noticed that we don't have this preview in this example, though we have it in all the others.